dhctl/pkg/kubernetes/actions/converge/converge.go

```diff
@@ -152,13 +154,114 @@ func (r *Runner) isSkip(phase Phase) bool {
 
 func (r *Runner) RunConverge() error {
 	if r.lockRunner != nil {
-		return r.lockRunner.Run(r.converge)
-	} else {
-		return r.converge()
+		err := r.lockRunner.Run()
+		if err != nil {
+			return fmt.Errorf("failed to start lock runner: %w", err)
+		}
+	}
+
+	return r.converge()
+}
+
+func (r *Runner) converge() (err error) {
+	state, err := r.stateStore.GetState()
```

Comment:
hide package here. I think need to rename it

Reply:
Fixed